docs/supported-clients.md

```diff
@@ -35,8 +36,9 @@ Wildebeest also provides a read-only web client in your instance URL, where you
 [9]: https://github.com/elk-zone/elk
 [10]: https://tooot.app/
 [11]: https://github.com/tooot-app
-[12]: https://testflight.apple.com/join/66c1wW8y
+[12]: https://testflight.apple.com/join/ejJ70WEq
```

Comment:
why does this link change? can't we use a stable link?

Reply:
I'm not too sure - most of the time TestFlight links are stable  but the developer might've recreated the app on Apple's side (I only found the new one by looking through the developer's recent Mastodon toots where they said the old link's build won't be updated)

Reply:
It says `This beta isn't accepting any new testers right now.`, the previous link says `To join the Mammoth: for Mastodon beta, open the link on your iPhone, iPad, or Mac after you install TestFlight.`

can you please revert that change? maybe we can ask the authors?

Reply:
Alright I'll revert it now!